packages/config/config/devices/0x0072/pe653.json

```diff
@@ -1,18 +1,39 @@
 {
-	"manufacturer": "Intermatic",
-	"manufacturerId": "0x0005",
-	"label": "PE653",
-	"description": "Pool Control",
+	"manufacturer": "Intermatic 0x0072 WaterTemp",
+	"manufacturerId": "0x0072",
+	"label": "PE653 0x0072 WaterTemp",
+	"description": "Pool Control 0x0072 WaterTemp",
```

Comment:
Just edit the manufacturer ID, keep the rest as-is. Unless the device is marketed under a new name?